build/configs/rtl8720e/hello/defconfig

```diff
@@ -262,7 +262,7 @@ CONFIG_ARCH_BOARD_HAVE_FLASH=y
 CONFIG_FLASH_MINOR=0
 CONFIG_FLASH_PART_SIZE="32,28,16,4,432,3068,3068,512,1024,8,"
 CONFIG_FLASH_PART_TYPE="none,none,none,none,none,kernel,kernel,smartfs,none,bootparam,"
-CONFIG_FLASH_PART_NAME="bl,reserved,ftl,system_data,ss,kernel,kernel,userfs,samsung_reserved,bootparam,"
+CONFIG_FLASH_PART_NAME="bl,reserved,ftl,system_data,ss,kernel,kernel,userfs,ss_reserved,bootparam,"
```

Comment:
Could you let me know why you modify this?

Reply:
If I enable CONFIG_MTD_PARTITION_NAMES, I will see "ERROR: Partition name is so long. Please make it smaller than 15". So I make it less than 15.

#ifdef CONFIG_MTD_PARTITION_NAMES
#define MTD_PARTNAME_LEN       15
#endif